packages/build-ui/src/components/button.rs

```diff
@@ -15,7 +15,10 @@ pub fn SecondaryButton(
     );
     rsx! {
         button {
-            class: "px-4.5 py-3 border border-white text-white bg-black  active:bg-white active:text-black hover:bg-white hover:text-black disabled:border-neutral-80 disabled:text-neutral-80 disabled:bg-btn-disabled ",
+            class: format!(
+                "px-4.5 py-3 border border-white bg-black active:bg-white active:text-black hover:bg-white hover:text-black disabled:border-neutral-80 disabled:text-neutral-80 disabled:bg-btn-disabled {}",
+                class,
+            ),
```

Comment:
```suggestion
            class: "px-4.5 py-3 border border-white bg-black active:bg-white active:text-black hover:bg-white hover:text-black disabled:border-neutral-80 disabled:text-neutral-80 disabled:bg-btn-disabled aria-condition:...",
             ,
            "aria-condition": if { } else { }
```

it is more efficient because it can loosely depend on rust code and reduce compilation time